src/review-form.ts

```diff
@@ -160,6 +210,35 @@ export class ReviewForm extends LitElement {
     `;
   }
 
+  private async setupRecaptcha(): Promise<void> {
+    this.recaptchaWidget = await this.recaptchaManager?.getRecaptchaWidget();
```

Comment:
Since `getRecaptchaWidget` returns a Promise and can error, I might do a try/catch block and set `this.recaptchaError = true` if we enter the `catch` block